apps/rush-lib/src/data/EnvironmentConfiguration.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+const ENV_VALUE_PREFIX: string = 'rush_';
+
+export enum EnvironmentValue {
+  TempDirectoryOverride = 'rush_tempDir'
```

Comment:
>TempDirectoryOverride [](start = 2, length = 21)

I would suggest to export this as part of the API surface for rush-lib, and use this as your place to write JSDoc for the variable.

Reply:
That doesn't really make sense in the new design with properties instead of an accessor function.

---
In reply to: [177928975](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177928975) [](ancestors = 177928975)